src/sparseml/pytorch/utils/sparsification_info/module_sparsification_info.py

```diff
@@ -15,25 +15,48 @@
 from typing import Iterable, Tuple
 
 import torch
-from pydantic import BaseModel
+from pydantic import BaseModel, Field
 
 from sparseml.pytorch.utils.sparsification_info.configs import (
-    SparsificationDistillation,
     SparsificationPruning,
     SparsificationQuantization,
     SparsificationSummaries,
 )
 
 
 class ModuleSparsificationInfo(BaseModel):
-    summary_info: SparsificationSummaries
-    pruning_info: SparsificationPruning
-    quantization_info: SparsificationQuantization
-    distillation_info: SparsificationDistillation
```

Comment:
Just wanted to double check that this field was removed

Reply:
Yes, it was expanded -  see lines 32-40